course_discovery/apps/edly/api/v1/serializers.py

```diff
@@ -0,0 +1,96 @@
+"""
+Serializers for Edly API.
+"""
+from course_discovery.apps.api.fields import StdImageSerializerField
+from rest_framework.serializers import (
+    ChoiceField,
+    IntegerField
+)
+from course_discovery.apps.course_metadata.models import (
+    Course,
+    Organization,
+    Program
+)
+from course_discovery.apps.core.models import Partner
+from course_discovery.apps.api.serializers import (
+    CourseSerializer,
+    CourseRunSerializer,
+    CorporateEndorsementSerializer,
+    DegreeSerializer,
+    EndorsementSerializer,
+    FAQSerializer,
+    MinimalPersonSerializer,
+    OrganizationSerializer,
+    ProgramSerializer,
+    SubjectSerializer,
+    VideoSerializer
+)
+
+
+class EdlyProgramSerializer(ProgramSerializer):
+    status = ChoiceField(choices=['active', 'unpublished'])
```

Comment:
I think its better if we define the choice field options in a separate variable/constant with proper formatting and use that here.

Reply:
We don't have huge list, that's why keeping it here looks better to me. What do you say?